cli/internal/helm/loader.go

```diff
@@ -146,21 +155,18 @@ func (i *ChartLoader) loadRelease(info chartInfo, helmWaitMode helm.WaitMode) (h
 
 	switch info.releaseName {
 	case ciliumInfo.releaseName:
-		values, err = i.loadCiliumValues()
+		values = ciliumVals[i.csp.String()]
```

Comment:
If we decide to remove the check for non-existent entries should we add a check that the map contains values for each `cloudprovider.Provider` value? I worry about e.g. foregtting to add a new CSP to the cilium values and not having a clear error output.